packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -1258,6 +1220,7 @@ export function TransferPanelMain({
             </p>
           )}
         </div>
+        <EstimatedGas chain={app.isDepositMode ? 'parent' : 'child'} />
```

Comment:
idk what good naming for that would be, but `chain` could be confused with an actual `Chain`.

some ideas:
- layer
- layerType
- chainType